### ISSUE TITLE
Unreadable text when using "Daytime" theme

### ISSUE THREAD

**Issue by seelchen** [TeamAmaze/AmazeFileManager#3993]
**Describe the bug**
When App theme is set to "Daytime" theme and the app is open while the time changes to after 6pm, the text becomes unreadable because the text color is changed to white (according to the time rules of the theme) while the background has not been updated to be dark.

**To Reproduce**
1. Open app before 6pm
2. Set app theme to "Daytime" theme
3. Close app settings
4. Keep app open until it is after 6pm 
5. Click on any directory
6. The text is unreadable 

**Expected behavior**
Either the background is also updated when it is 6pm or the text should also stay black and only change to white when the background is updated.

**Screenshots**
(the clicking seems very glitchy in the video because of the screenrecording)

https://github.com/TeamAmaze/AmazeFileManager/assets/74261221/1561dee3-f4f9-4709-9f98-984f9aeba43d

**Smartphone (please complete the following information):**
 - Device: emulated Pixel 7
 - OS: Android 13
 - Rooted: Yes
 - Version: built from branch `release/4.0`

**Comment by seelchen** [comment]
My proposal would be to use a [`ScheduledExecutorService`](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ScheduledExecutorService.html#scheduleAtFixedRate-java.lang.Runnable-long-long-java.util.concurrent.TimeUnit-) that updates the activity every 12 hours at 6am or 6pm. 
This solution could be a little bit overkill since I don't think that many users would keep the app open for over 12 hours. However, I think it would be better to let the Java implementation handle this and not write something custom.

**Comment by seelchen** [comment]
Yes, the issue doesn't persist after app restart. It only affects the activity that is open when the time changes to 6pm. If any other activity is opened afterwards, the correct background will be set and everything will be fine for another 12 hours.

**Comment by VishalNehra** [comment]
Damn talk about corner case XD
